### PR TITLE
should user quotation marks for ./arkmanager broadcast

### PR DESCRIPTION
**Pull request by lexat**
using $2 for broadcasttext in script -> 

./arkmanager broadcast hello world
will result in a broadcast with text "hello"
./arkmanager broadcast "hello world"
will take all to broadcast

### CODE REVIEW

**Comment by FezVrasta** [comment]
Then you don't need to escape the spaces, no?

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ manually updates ARK server
 broadcast a message to ARK server chat
 
 ```sh
-arkmanager boradcast here\ your\ message
+arkmanager boradcast "here\ your\ message"
```
Comment (FezVrasta) [inline README.md]: "here your message"

Comment (lexat) [inline README.md]: thats what i mean, but i cant change pull :D

Comment (FezVrasta) [inline README.md]: well, you can, you can just push on the branch of your pull request. by the way I'll fix it by myself this time.

**Pull request merged**